FF1Lib/LegendaryShops.cs

```diff
@@ -200,11 +218,30 @@ private int[] AllocateSlots()
 
 			int[] result = new int[] { 0, 0, 0, 0, 0 };
 
-			if (flags.LegendaryBlackShop ?? false) result[2] = Allocate(ref slots, 5);//Black
-			if (flags.LegendaryArmorShop ?? false) result[1] = Allocate(ref slots, 5);//Armor
-			if (flags.LegendaryWhiteShop ?? false) result[3] = Allocate(ref slots, 4);//White
-			if (flags.LegendaryWeaponShop ?? false) result[0] = Allocate(ref slots, 3);//Weapon
-			if (flags.LegendaryItemShop ?? false) result[4] = Allocate(ref slots, 3);//Item
+			if (flags.LegendaryBlackShop ?? false)

```

Comment:
Thats 5 lines inflated to 24.

Reply:
yeah, this is due the the default rules and is up for discussion, in general I do think oneliners can be needlessly confusing when being more verbose only takes up more line -- its not like we need to ration lines. though I do agree that in specific cases oneliners can be better, as showcased here.